### PR TITLE
DEV-3884-add Subject: Rect Perimeter exercise

### PR DESCRIPTION
**Pull request by mohamedLazyBob**
resolves 01-edu/public#1149

### CODE REVIEW

File: subjects/rectperimeter/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# rectperimeter
+
+### Instructions
+
+write a function that accepts two integer arguments, representing the two sides length of a rectangle and returning the perimeter of the rectangle.
+
+your function should return `-1` if one of the arguments is negative.
```
Comment (Hamzaelkhatri) [inline subjects/rectperimeter/README.md:7]: remove Code-quotes from 1, 
code-quotes only for : 
- function names
- libraries
- data types
- etc.

**Review by Hamzaelkhatri — CHANGES_REQUESTED** [review]

**Review by zainabdnaya — COMMENTED** [review]
Every thing is good

File: subjects/rectperimeter/README.md (hunk 1 above)
Comment (mohamedLazyBob) [reply subjects/rectperimeter/README.md]: Done

File: subjects/rectperimeter/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# rectperimeter
```
Comment (davidrobert99) [inline subjects/rectperimeter/README.md:1]: Missing a "#", should be "##"

File: subjects/rectperimeter/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+### Expected function
+
+```go
+func RectPerimeter(l, w int) int {
+
+}
+```
+
+### Usage
+
```
Comment (davidrobert99) [inline subjects/rectperimeter/README.md:18]: Missing "Here is a possible program to test your function:"

**Review by davidrobert99 — CHANGES_REQUESTED** [review]

File: subjects/rectperimeter/README.md (hunk 3 above)
Comment (jotapero) [reply subjects/rectperimeter/README.md:18]: fixed

**Review by jotapero — APPROVED** [review]
Looks Good :alien:

File: subjects/rectperimeter/README.md (hunk 2 above)
Comment (jotapero) [reply subjects/rectperimeter/README.md]: fixed

File: subjects/rectperimeter/README.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+import (
+	"fmt"
+
+	"piscine"
+)
+
+func main() {
+	fmt.Println(piscine.RectPerimeter(10, 2))
+	fmt.Println(piscine.RectPerimeter(434343, 898989))
+}
```
Comment (tiagocollot) [inline subjects/rectperimeter/README.md:33]: We should add one case for this condition: "If one of the arguments is negative it should return `-1`."

File: subjects/rectperimeter/README.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+```
+
+And its output:
+
+```console
+$ go run .
+24$
+2666664$
+$
```
Comment (tiagocollot) [inline subjects/rectperimeter/README.md:42]: Same as above.

**Review by tiagocollot — CHANGES_REQUESTED** [review]

File: subjects/rectperimeter/README.md (hunk 4 above)
Comment (jotapero) [reply subjects/rectperimeter/README.md:34]: fixed

File: subjects/rectperimeter/README.md (hunk 5 above)
Comment (jotapero) [reply subjects/rectperimeter/README.md:44]: fixed

Comment (jotapero) [reply subjects/rectperimeter/README.md:44]: fixed

**Review by jotapero — APPROVED** [review]

**Pull request merged**